x-pack/plugins/observability_solution/observability_ai_assistant/server/config.ts

```diff
@@ -11,6 +11,7 @@ export const config = schema.object({
   enabled: schema.boolean({ defaultValue: true }),
   modelId: schema.maybe(schema.string()),
   scope: schema.maybe(schema.oneOf([schema.literal('observability'), schema.literal('search')])),
+  enableKnowledgeBase: schema.boolean({ defaultValue: true }),
```

Comment:
what's the plan here for non-serverless?

Reply:
always on